### PR TITLE
Add graingert-wow v1.2.0 with npm auto-update

### LINKED ISSUE #7866: [Request] wow.js (MIT forever free)
**Library name:** wow.js
**Git repository url:** https://github.com/graingert/WOW.git
**npm package url(optional):** https://www.npmjs.com/package/wow.js
**License(s):** MIT
**Official homepage:** https://wow.js.org
**Wanna say something? Leave message here:** This replaces the existing https://cdnjs.com/libraries/wow that is now GPL/shareware.

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by x09326**
PR for #7866 
@dannyxx001 please help me review it, thanks.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/graingert/WOW
 * Official website (optional, not the repository):https://graingert.co.uk/WOW/
 * NPM package url (optional):https://www.npmjs.com/package/wow.js
 * GitHub / Bitbucket popularity (required):
   - Count of stars:1
   - Count of watchers:1
   - Count of forks:1309
 * NPM download stats (optional):
   - Downloads in the last day:29
   - Downloads in the last week:294
   - Downloads in the last month:994

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7866, cc @graingert

### CODE REVIEW

File: ajax/libs/graingert-wow/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "graingert-wow",
+  "filename": "wow.min.js",
+  "title": "WOW",
```
Comment (dannyxx001) [inline ajax/libs/graingert-wow/package.json]: I think we don't need this field.

Comment (PeterDaveHello) [inline ajax/libs/graingert-wow/package.json]: hmmm ... we can preserve this.

Comment (dannyxx001) [inline ajax/libs/graingert-wow/package.json]: Sorry for late response. @x09326  Because that's OK, we can rebase first and then call Peter.

**Comment by x09326** [comment]
@dannyxx001 
I have rebased it.
Please help me review it again, thanks.

**Comment by PeterDaveHello** [comment]
We have wow here: https://cdnjs.com/libraries/wow, and you are adding a fork of it, you shoule name it `graingert-wow`.

**Comment by graingert** [comment]
@PeterDaveHello you've already renamed it. I'd rather it be mit-wow though

**Comment by x09326** [comment]
@PeterDaveHello 
Do you mean modify the ```"title"``` of **package.json**?

**Comment by PeterDaveHello** [comment]
@x09326 Name.

**Comment by PeterDaveHello** [comment]
Oops, sorry, seems I missed something.

**Comment by x09326** [comment]
Um....but the name is ```graingert-wow```.

**Comment by PeterDaveHello** [comment]
Yeah, I'll merge this now.

**Pull request merged**